File: app/src/main/java/com/peakmain/basicui/activity/home/recylcer/ItemTouchHelper/GridItemTouchHelper.java
package com.peakmain.basicui.activity.home.recylcer.ItemTouchHelper;

import android.support.v7.widget.RecyclerView;

import com.peakmain.basicui.activity.home.recylcer.bean.GroupBean;
import com.peakmain.ui.recyclerview.itemTouch.BaseItemTouchHelper;

import java.util.List;

/**
 * author ：Peakmain
 * createTime：2020/3/25
 * mail:2726449200@qq.com
 * describe：
 */
public class GridItemTouchHelper extends BaseItemTouchHelper<GroupBean> {


    public GridItemTouchHelper(RecyclerView.Adapter adapter, List<GroupBean> data) {
        super(adapter, data);
    }
}

File: app/src/main/java/com/peakmain/basicui/activity/home/recylcer/itemTouchHelper/GridItemTouchHelper.kt
package com.peakmain.basicui.activity.home.recylcer.itemTouchHelper

import androidx.recyclerview.widget.RecyclerView
import com.peakmain.basicui.activity.home.recylcer.bean.GroupBean
import com.peakmain.ui.recyclerview.adapter.ViewHolder
import com.peakmain.ui.recyclerview.itemTouch.BaseItemTouchHelper

/**
 * author ：Peakmain
 * createTime：2020/3/25
 * mail:2726449200@qq.com
 * describe：
 */
class GridItemTouchHelper(adapter: RecyclerView.Adapter<ViewHolder>?, data: List<GroupBean?>?) : BaseItemTouchHelper<GroupBean?>(adapter, data)